core/services/job/models.go

```diff
@@ -878,6 +878,7 @@ func (w *WorkflowSpec) Validate() error {
 		return fmt.Errorf("%w: incorrect length for id %s: expected %d, got %d", ErrInvalidWorkflowID, w.WorkflowID, workflowIDLen, len(w.WorkflowID))
 	}
 
+	w.WorkflowOwner = strings.TrimPrefix(w.WorkflowOwner, "0x")
```

Comment:
Shouldn't we then just require "0x" to always be there? Owner is never set by the user, always by tooling, so it shouldn't cause any unnecessary mistakes.

Reply:
we can change it to only accept the prefixed value latter. all our tests use non-0x prefixes, which is why i missed it to being with